news/2019-04-10-bn-and-qat-restructure.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "BN and QAT Restructure"
+date: 2019-04-10 23:30:00 +0000
+---
+
+Changes to the ranking process are nigh! With recent discussions about [changes to the Quality Assurance Team and Beatmap Nominators](https://osu.ppy.sh/community/forums/topics/862002), we are proud to present to you the following changes.
+
+![](/wiki/shared/news/banners/beatmap-management.jpg)
+
++ The QAT is gone and has been replaced by the new [Nominator Administration Team](https://osu.ppy.sh/groups/7).
++ The Beatmap Nominators (BN) are responsible for disqualifying beatmaps and mediating vetoes from now on, in addition to their task of pushing beatmaps towards the ranked category.
++ The Nominator Administration Team (NAT) are the moderators of the Beatmap Nominators. They are responsible for BN Applications, creating tests for the applicants and for evaluating the current members. They will still have their moderation privileges although it is not their primary focus. 
++ Beatmaps that have been vetoed will be mediated by a random jury made up of members of the BN. The NAT will oversee these mediations. 
++ Probationary Beatmap Nominators now have their own usergroup. These members will have less features compared to the rest of the BNs.
```

Comment:
maybe just phrase as "Probationary Beatmap Nominators won't be able to \<do x things\>", the usergroup won't be public so it's kinda useless information

probably works either way though.

Reply:
I added that information in the next paragraph. Was told that even if probation has been for a while, since it's an actual osu-web change it should be mentioned in the news post, and to give as much info about it as we can.

Reply:
then it's also backwards, the new usergroup is for non-probationary bns (could still change, waiting for review about that)

Reply:
```suggestion
+ The current Beatmap Nominator usergroup will be used for probationary Beatmap Nominators. A new usergroup will be made for regular members of the Beatmap Nominators which will grant access to the disqualification button.
```

Reply:
I don't think it's a good idea to explain implementation details in a news post, if people are curious they can easily find details in osu-web or just by poking around on the website

anyway if this stays you'll have to update it again https://github.com/ppy/osu-web/pull/4402

Reply:
Should already be updated (different to what I suggested above). Got criticism from higher up that the news post lacked detail, so here we are.

Reply:
yea nvm I think i was looking at an intermediate version it is correct now

ig it makes sense to include this info now that both groups are intended to be public, but next time ask them to comment here so we are not changing things without a (public) reason